schema/compositor.cpp

```diff
@@ -97,6 +99,12 @@ Compositor::Type Compositor::type() const
     return d->mType;
 }
 
+QString Compositor::typeName() const
+{
+    QMetaEnum metaEnum = QMetaEnum::fromType<Compositor::Type>();
```

Comment:
I assume adding this method is a groundwork for some further fixes, so at the moment I am not sure how often will it be called. But I think we should rather move the metaEnum to some static m_typeMetaEnum member to be initialised only once.

Reply:
This intended to enhance verbose mode in kxml_compiler here https://github.com/cornelius/kode/blob/master/kxml_compiler/parserxsd.cpp#L146